docs/CHANGELOG.asciidoc

```diff
@@ -19,3 +19,255 @@ This section summarizes the changes in each release.
+====
+      
+[discrete]
+[[breaking-90511]]
+.Removed default support for TLS v1.0 and v1.1
+[%collapsible]
+====
+*Details* +
+The default support for TLS v1.0 and v1.1 has been removed. For more information, refer to {kibana-pull}90511[#90511]
+
+*Impact* +
+To enable support, set the environment variable to `NODE_OPTIONS=--tls-min-1.0`.
+====
+           
+[discrete]
+[[breaking-74424]]
+.Removed support for sysv init
+[%collapsible]
+====
+*Details* +
+Systems that don't have `service` aliased to use kibana.service are unable to use `service start kibana`. For more information, refer to {kibana-pull}74424[#74424]
+
+*Impact* +
+If your system doesn't have `service` aliased to use kibana.service, use `systemctl start kibana.service`.
+====
+       
+[discrete]
+[[breaking-42353]]
+.Disabled response logging as a default
+[%collapsible]
+====
+*Details* +
+By default, responses are not logged. Previously, responses were logged if `logging.json` was set to `true`, `logging.dest` was specified, or a TTY was detected. For more information, refer to {kibana-pull}42353[#42353]
+
+*Impact* +
+To log responses, set `logging.events.response=*` in kibana.yml.
+====
+
+[float]
+[[reporting-changes-8.0.0-alpha1]]
+==== Reporting changes
+
+[discrete]
+[[breaking-64853]]
+.Task Manager integration for Reporting
+[%collapsible]
+====
+*Details* +
+The ES-Queue subservice of *Reporting* has been removed and replaced with *Task Manager* integration. For more information, refer to {kibana-pull}64853[#64853]
+
+*Impact* +
+Before you upgrade, make sure there are no pending reporting jobs. Any pending reporting jobs will remain in the pending state.
+====
+      
+[discrete]
+[[breaking-52539]]
+.Removed legacy Reporting job params compatibility shim
+[%collapsible]
+====
+*Details* +
+*Reporting* is no longer compatible with POST URL snippets generated with {kib} 6.2.0 and earlier. For more information, refer to {kibana-pull}52539[#52539]
+
+*Impact* +
+If you use POST URL snippets to automatically generate PDF reports, regenerate the POST URL strings.
+====
+
+[float]
+[[rest-api-changes]]
+==== Security changes
+
+[discrete]
+[[breaking-47929]]
+.Removed `/api/security/v1/saml` route
+[%collapsible]
+====
+*Details* +
+The `/api/security/v1/saml` route has been removed and is reflected in the kibana.yml `server.xsrf.whitelist` setting, {es}, and the Identity Provider SAML settings. For more information, refer to {kibana-pull}47929[#47929]
+
+*Impact* +
+Use the `/api/security/saml/callback` route.
+====
+
+[discrete]
+[[breaking-41700]]
+.Reject legacy browsers by default
+[%collapsible]
+====
+*Details* +
+To provide the maximum level of protection for most installations, the csp.strict config is now enabled by default. Legacy browsers, such as IE11, are unable to access {kib} unless explicitly enabled. For more information, refer to {kibana-pull}41700[#41700]
```

Comment:
We actually stopped supporting IE11 a little while ago, so I think it's worth adding a note that this is no longer supported.
```suggestion
To provide the maximum level of protection for most installations, the csp.strict config is now enabled by default. Legacy browsers not supported by Kibana, such as IE11, are unable to access {kib} unless explicitly enabled. All browsers officially supported by Kibana do not have this issue. For more information, refer to {kibana-pull}41700[#41700]
```